app/assets/stylesheets/govuk_admin_template/_flash.scss

```diff
@@ -0,0 +1,13 @@
+.alert {
+  &.flash-notice {
```

Comment:
I'd prefer to use the Bootstrap classes as they are, rather than extend them. It avoids having to document them in a style guide. It also keeps the Bootstrap docs themselves as a canonical guide.